controllers/sriovoperatorconfig_controller.go

```diff
@@ -70,8 +71,8 @@ func (r *SriovOperatorConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 			defaultConfig.SetNamespace(namespace)
 			defaultConfig.SetName(DEFAULT_CONFIG_NAME)
 			defaultConfig.Spec = sriovnetworkv1.SriovOperatorConfigSpec{
-				EnableInjector:           func() *bool { b := true; return &b }(),
-				EnableOperatorWebhook:    func() *bool { b := true; return &b }(),
+				EnableInjector:           func() *bool { b := enableAdmissionController; return &b }(),
```

Comment:
This may not be needed since `ENABLE_ADMISSION_CONTROLLER` takes precedence when rendering webhook objects.

Reply:
If we really want to set the right flags in default OperatorConfig, it will be in `createDefaultOperatorConfig` function and here. but user can still update these flags to true without it taking effects.

Reply:
Updated the `createDefaultOperatorConfig`